src/linter.js

```diff
@@ -1,9 +1,10 @@
-import { extname } from 'path';
+import path from 'path';
 
 import columnify from 'columnify';
 import chalk from 'chalk';
 import Dispensary from 'dispensary';
 import { oneLine } from 'common-tags';
+import cheerio from 'cheerio';
```

Comment:
Nit, if we are going to keep the "HTML parsing" part of these changes, let's move this import after the one for `chalk` (it seems to me that the other imports groups are being kept in alphabetic order).